src/cf-forms.less

```diff
@@ -56,15 +60,32 @@
         <label class="form-label-header">
             Form label header
         </label>
+    - name: Form label helper text
+      markup: |
+        <label class="form-label-helper-text">
+            Form label helper text
+        </label>
   tags:
     - cf-forms
 */
 
 .form-label-header {
-    .h5();
+    .h4();
     margin-bottom: unit(10px / @font-size, em);
 }
 
+.form-label-helper-text {
+    .micro-copy();
+    color: @label-helper;
+    margin-bottom: unit(10px / 14px, em);
+}
+
+.form-label-error {
+    .error + & {
+      margin-top: unit(15px / 16px, em);      
```

Comment:
- inconsistent indentation
- Please use `@base-font-size-px` instead of `16px`
- Let's unnest this into a single `.error + .form-label-error` declaration until such time as nesting would be beneficial.